postgres-persistence/build.gradle

```diff
@@ -0,0 +1,19 @@
+dependencies {
+    compile project(':conductor-core')
+    compile "com.google.inject:guice:${revGuice}"
+
+    compile "commons-io:commons-io:${revCommonsIo}"
+    compile "org.postgresql:postgresql:42.2.6"
```

Comment:
Please move version to `versionsOfDependencies.gradle`

Reply:
ok; updated